### PR TITLE
Update for preview9

### PR DESCRIPTION
**Pull request by jsakamoto**
## Update for preview8

- Updated / adjusted dependencies.
- `WebAssemblyUriHelper` was gone, therefore, I changed the way to retrieve base URI. (invoke JavaScript interop, instead.)
- Changed the class name from `DotNetObjectRef` to `DotNetObjectReference`.

### CODE REVIEW

File: src/BlazorSignalR/BlazorSignalR.csproj [hunk 1]
```diff
@@ -8,14 +8,14 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);IncludeP2POutput</TargetsForTfmSpecificBuildOutput>
-    <Version>0.9.0-blazor-3.0.0-preview8.19405.7</Version>
+    <Version>0.10.0-blazor-3.0.0-3.0.0-preview9.19424.4</Version>
```
Comment (jsakamoto) [inline src/BlazorSignalR/BlazorSignalR.csproj]: Oh! This is my mistake.
It should be "0.10.0-blazor-3.0.0-preview9.19424.4", not "0.10.0-blazor-3.0.0-3.0.0-preview9.19424.4".

**Comment by csnewman** [comment]
Overlaps with #36 going to need to take a look into the two PRs a bit more.

**Comment by dazinator** [comment]
@csnewman I took at look at this PR and competing one #36 

`NavigationManager` on PR #36 is the correct way to go about things in preview 9 rather than introducing js interop, other than that these two PR's are pretty similar. Therefore i'd vote (sorry @jsakamoto) to abandon this PR in favour of #PR 36

**Comment by jsakamoto** [comment]
@dazinator Don't worry! I agree your opinion.

This PR is just my favorite way.

I sometimes prefer keep compatibility with some hack, rather than formal way with breaking change. 
That's it. 😄

**Comment by csnewman** [comment]
I agree with @dazinator. Thanks for creating this PR @jsakamoto, however I'm going to go with the other PR for the above listed reasons.

**Comment by jsakamoto** [comment]
@csnewman No problem!

Thank you for your kindly reply.

And, I appreciate to you that you publish these great products as a open source!

**Pull request closed without merging**